package.json

```diff
@@ -57,10 +57,10 @@
     "test": "nyc --reporter=lcov --reporter=html --reporter=text-summary npm run test-bare",
     "lint": "eslint . --cache",
     "docs": "node scripts/generate-readme.js",
-    "integration-test-cleanup": "node scripts/integration-test-cleanup.js",
-    "packaging-integration-test": "jest --maxWorkers=5 packaging-suite",
-    "simple-integration-test": "jest --maxWorkers=5 simple-suite",
-    "complex-integration-test": "jest --maxWorkers=5 integration",
+    "integration-test-run-package": "jest --maxWorkers=5 integration-package",
+    "integration-test-run-basic": "jest --maxWorkers=5 integration-basic",
+    "integration-test-run-all": "jest --maxWorkers=5 integration-all",
```

Comment:
Why do we imply `--maxWorkers`?

Reply:
We had AWS issues when running too many tests in parallel. Those issues were related to AWS limitations. Not sure if they've been loosened in the meantime.

Reply:
We possibly need to solve it at tests layer (e.g. to retry after retryable error).. as it may still occur with concurrent builds which will definitely happen in case of _basic_ tests.

Still we may leave it until we indeed have such failures